src/Host/Client/Impl/Definitions/IRSessionProvider.cs

```diff
@@ -39,5 +41,10 @@ public interface IRSessionProvider : IDisposable {
         /// <param name="name">Name of the broker. Will be displayed in REPL.</param>
         /// <param name="path">Either a local path to the R binary or a URL to the broker.</param>
         Task<bool> TrySwitchBrokerAsync(string name, string path = null);
+
+        /// <summary>
+        /// Displays broker machine and process information
+        /// </summary>
+        void DisplayBrokerInformation();
```

Comment:
`PrintBrokerInformation` maybe? Cause it doesn't really display anything - it prints the information to the callback